wrphttp/headers.go

```diff
@@ -57,14 +80,17 @@ var (
 func getMessageType(h http.Header) wrp.MessageType {
 	value := h.Get(MessageTypeHeader)
 	if len(value) == 0 {
```

Comment:
```suggestion
	if value == "" {
```

Reply:
The rest of the code around this is also using `len(string) == 0`.  While that seems less than ideal, I think consistency is better in this case.  We can/should go through as a separate PR and change this pattern in the future if we want; but consistently.

Reply:
Yeah, I was just matching what was already there, but I'll update to your suggestion! It's more readable that way

Reply:
As Wes pointed out, the rest of the around this also uses `len(string) == 0` and consistency is better at the moment. We will leave this conditional as it is.